Updates/5.0.0-rc2.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\TagManager;
+
+use Piwik\Plugins\TagManager\Template\Variable\MatomoConfigurationVariable;
+use Piwik\Plugins\TagManager\UpdateHelper\NewVariableParameterMigrator;
+use Piwik\Updater;
+use Piwik\Updater\Migration;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+use Piwik\Updates as PiwikUpdates;
+
+/**
+ * Update for version 5.0.0-rc2.
+ */
+class Updates_5_0_0_rc2 extends PiwikUpdates
```

Comment:
@EPNW-Eric [5.0.0-rc2](https://github.com/matomo-org/matomo/releases/tag/5.0.0-rc2) was released 4 days ago can you change it to rc3 ?

Reply:
Will do 👍